src/Adapter/Product/Combination/Repository/CombinationRepository.php

```diff
@@ -106,6 +106,23 @@ public function get(CombinationId $combinationId): Combination
         return $combination;
     }
 
+    public function getProductId(CombinationId $combinationId): ProductId
```

Comment:
I think it's quite explicit since we are in the `CombinationRepository` and, most of all, there is only one parameter which is typed with a VO ProductId, if it was a simple int I would agree that it needs more explicit naming but here the typing says it all

Just like the following function `getDefaultCombinationId` doesn't need to be named `getDefaultCombinationIdByProductId` it is a but redundant

Reply:
different habits .... but I guess to know how you will name a future method that get the product id by an another dimension. 
getProductId(FooId $fooId) ?